_includes/head.html

```diff
@@ -35,12 +34,8 @@
     })();
   </script>
 
-  <!-- include html5shim per Explorer 8 -->
-  <script src="{{ '/assets/toolkit/vendor/modernizr.js' | relative_url }}"></script>
```

Comment:
by removing this we drop the support for IE8?

Reply:
Bootstrap4 is not ie8 compliant

Reply:
ok, not a problem for me, we just need to agree that the new version of the website will not be compatibile with IE8. @rasky is this OK for you?